redhat/overlays/cosign/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#Build stage
+FROM golang:1.20 AS build-env
```

Comment:
This should use a Red Hat image. E.g. https://github.com/securesign/trillian/blob/fd0b9bb3dec783f1c2012fb036364d5ca68b48c3/redhat/overlays/log_server/Dockerfile#L1

Reply:
No problem, I've also noticed that you have made changes in the securesign/rekor repo to how the branching is set up should I go ahead and incorporate those changes here as well as in the image factory ?

Reply:
Yes - I would use the `redhat-v2.1.1` branch in image-factory to pin this to the latest release.